Controls/main.c

```diff
@@ -0,0 +1,17 @@
+#include "MotorController.h"
+#include "stdlib.h"
+#include <time.h>
+
+void delay(int mseconds){
+    clock_t start_time = clock();
+    while (clock()<start_time+mseconds){
```

Comment:
If you're going to use this clock() function for the delay, you need to make sure to divide by CLOCKS_PER_SEC to get an amount of seconds instead of clock ticks. This right now is only delaying mseconds amount of clock ticks.